### DISCUSSION TITLE
Magiclysm: Magical Broom for flying

### DISCUSSION THREAD

**Discussion by fgdfgfthgr-fox** [CleverRaven/Cataclysm-DDA#46609] [Ideas]
Since we have helicopters now, it seems rather interesting to add a vehicle part called "magical broom", which provide vehicle flying ability (only when it's very light or you use multiple brooms at the same time!). But instead using engine power or electricity, it burns mana dust/crystallized mana.
Consider how op it's to be flying, it has to be either not craftable or require really high skills to craft. Or it requires a profession proficiency to drive it and so on...
"A broom gained flying ability due to magic. Still a terrible weapon for fighting, not so terrible for flighting."
@KorGgenT

**Comment by KorGgenT** [comment]
sounds very hard to balance properly, as you said flying is extremely powerful.

**Reply by fgdfgfthgr-fox** [reply]
What about that you require a profession proficiency that's either very hard to learn or only accessible during character creation to drive(and make) a magical broom? Like what cdda did to helicopters.

**Reply by actual-nh** [reply]
As well, add the REQUIRES_BALANCE flag (like rollerblades).

**Comment by Altogolik** [comment]
A broom is not cool, I want a flying motorcycle (developed by a technomagician)

**Reply by JReese3** [reply]
If it flies, who cares what the part is that makes it happen.  While it would be cool to have it be some super-science magic unobtanium drop from a dragon...  Enchanted broomsticks with say, the carry capicity of a horse, would be cool as well.
